spec/takeaway_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  subject(:takeaway) {described_class.new(double(:text, send: nil))}
+  let(:menu) {"starter : 5\nmains : 10\ndesert : 3\n"}
+
+  before do
+
+  end
+
+  describe 'looking at a list of dishes with prices' do
+
+    it 'returns a list of dishes with price' do
+    expect{takeaway.show_dishes}.to output(menu).to_stdout
+    end
+
+  end
+
+  describe '#order' do
+
+    it 'can take multiple arguments' do
+      expect{takeaway.order("starter", 1, "mains", 2, "desert", 3, 34)}.not_to raise_error
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)